media-libs/openimageio/openimageio-1.8.17.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
```

Comment:
Try to change this to `PYTHON_COMPAT=( python{2_7,3_{4,5,6,7}} )` please. In case `repoman full` complains about dependencies missing python3_7 support, revert it.